src/Resources/Locales/es.json

```diff
@@ -1312,7 +1312,7 @@
         "remove": "Quitar gemelo de serie temporal",
         "property": "Propiedad",
         "propertyTypeCastLabel": "Convertir el tipo de propiedad en número",
-        "add": "Agregar gemelo de serie temporal"
+        "add": "Agregar series temporales"
```

Comment:
let's make sure to regen all the locales. It looks like there are more changes to en-us than there are to some of these other files.

Reply:
ahh this is weird, will check why the other keys didn't go through